src_assets/common/assets/web/configs/tabs/audiovideo/DisplayDeviceOptions.vue

```diff
@@ -5,41 +5,123 @@ import PlatformLayout from '../../../PlatformLayout.vue'
+                  </div>
+                  <input type="text" class="form-control" id="dd_manual_resolution" placeholder="2560x1440"
+                         v-model="config.dd_manual_resolution" />
+                </div>
+              </div>
+
+              <!-- Refresh rate option -->
+              <div class="mb-3" v-if="config.dd_configuration_option !== 'disabled'">
+                <label for="dd_refresh_rate_option" class="form-label">
+                  {{ $t('config.dd_refresh_rate_option') }}
+                </label>
+                <select id="dd_refresh_rate_option" class="form-select" v-model="config.dd_refresh_rate_option">
+                  <option value="disabled">{{ $t('config.dd_refresh_rate_option_disabled') }}</option>
+                  <option value="automatic">{{ $t('config.dd_refresh_rate_option_automatic') }}</option>
+                  <option value="manual">{{ $t('config.dd_refresh_rate_option_manual') }}</option>
+                </select>
+
+                <!-- Manual refresh rate -->
+                <div class="mt-2 ps-4" v-if="config.dd_refresh_rate_option === 'manual'">
+                  <div class="form-text">
+                    {{ $t('config.dd_refresh_rate_option_manual_desc') }}
+                  </div>
+                  <input type="text" class="form-control" id="dd_manual_refresh_rate" placeholder="59.9558"
+                         v-model="config.dd_manual_refresh_rate" />
+                </div>
+              </div>
+
+              <!-- HDR option -->
+              <div class="mb-3" v-if="config.dd_configuration_option !== 'disabled'">
+                <label for="dd_hdr_option" class="form-label">
+                  {{ $t('config.dd_hdr_option') }}
+                </label>
+                <select id="dd_hdr_option" class="mb-3 form-select" v-model="config.dd_hdr_option">
+                  <option value="disabled">{{ $t('config.dd_hdr_option_disabled') }}</option>
+                  <option value="automatic">{{ $t('config.dd_hdr_option_automatic') }}</option>
+                </select>
+                <div class="form-check">
+                  <!-- HDR toggle -->
+                  <label for="dd_wa_hdr_toggle" class="form-check-label">{{ $tp('config.dd_wa_hdr_toggle') }}</label>
+                  <input type="checkbox" class="form-check-input" id="dd_wa_hdr_toggle" true-value="true" false-value="false" v-model="config.dd_wa_hdr_toggle" />
+                  <div class="form-text">{{ $tp('config.dd_wa_hdr_toggle_desc') }}</div>
+                </div>
+              </div>
+
+              <!-- Config revert delay -->
+              <div class="mb-3" v-if="config.dd_configuration_option !== 'disabled'">
+                <label for="dd_config_revert_delay" class="form-label">
+                  {{ $t('config.dd_config_revert_delay') }}
+                </label>
+                <input type="text" class="form-control" id="dd_config_revert_delay" placeholder="10000"
```

Comment:
```suggestion
                <input type="text" class="form-control" id="dd_config_revert_delay" placeholder="3000"
```

Placeholders should match the default value whenever possible.